docs/tutorials/curve_analysis.rst

```diff
@@ -296,50 +296,50 @@ or elimination of outliers for better fitting.
 3. Fitting
 ^^^^^^^^^^
 
-Curve analysis calls :meth:`_run_curve_fit` method which is the core functionality of the fitting.
-The another method :meth:`_generate_fit_guesses` is internally called to
+Curve analysis calls the :meth:`_run_curve_fit` method, which is the core functionality of the fitting.
+Another method :meth:`_generate_fit_guesses` is internally called to
 prepare the initial guess and parameter boundary with respect to the formatted data.
-A developer usually override this method to provide better initial guess
+Developers usually override this method to provide better initial guesses
 tailored to the defined fit model or type of the associated experiment.
 See :ref:`curve_analysis_init_guess` for more details.
-A developer can also override the entire :meth:`_run_curve_fit` method to apply
-custom fitting algorithms. This method must return :class:`.CurveFitResult` dataclass.
+Developers can also override the entire :meth:`_run_curve_fit` method to apply
+custom fitting algorithms. This method must return a :class:`.CurveFitResult` dataclass.
 
 4. Post processing
 ^^^^^^^^^^^^^^^^^^
 
-Curve analysis runs several postprocessing against to the fit outcome.
-It calls :meth:`._create_analysis_results` to create :class:`.AnalysisResultData` class
-for the fitting parameters of interest. A developer can inject a custom code to
+Curve analysis runs several postprocessing against the fit outcome.
+It calls :meth:`._create_analysis_results` to create the :class:`.AnalysisResultData` class
+for the fitting parameters of interest. A developer can inject custom code to
 compute custom quantities based on the raw fit parameters.
 See :ref:`curve_analysis_results` for details.
-Afterwards, the analysis draws several curves in the Matplotlib figure.
+Afterwards, the analysis draws several curves in the figure.
 Users can set a custom plotter in :class:`.CurveAnalysis` classes, to customize
-figures, by setting the :attr:`~.CurveAnalysis.plotter` attribute.
-Finally, it returns the list of created analysis results and Matplotlib figure.
+figures, by setting the :attr:`~.CurveAnalysis.plotter` attribute (see documentation of the visualization module).
+Finally, it returns the list of created analysis results and a figure.
```

Comment:
How about this?

```
Afterwards, figure plotting is handed over to the :doc:`Visualization </tutorials/visualization>` module via
the :attr:`~.CurveAnalysis.plotter` attribute, and a list of created analysis results and figures are returned.
```

Reply:
It's very good, but a question: earlier the document says:
```
Finally, the analysis outputs a set of :class:`.AnalysisResultData` entries
for important fit outcomes along with a single figure of the fit curves
with the measured data points.
```
So, do we return `a list of created analysis results and figures` or `a list of created analysis results and a figure`?

Reply:
Done